crates/spirv-std/Cargo.toml

```
@@ -11,8 +11,8 @@ repository.workspace = true
 spirv-std-types.workspace = true
 spirv-std-macros.workspace = true
 bitflags = "1.2.1"
-num-traits = { version = "0.2.14", default-features = false, features = ["libm"] }
-glam = { version = "0.22", default-features = false, features = ["libm"] }
+num-traits = { version = "0.2.15", default-features = false, features = ["libm"] }
+glam = { version = "0.24", default-features = false, features = ["libm"] }
```

```suggestion
glam = { version = ">=0.22, <0.25", default-features = false, features = ["libm"] }
```
(kind of annoying to specify `0.25` but I'm worried `<=0.24` would exclude patch releases)

---

Oh I'm wrong, `<=0.24` will work with `0.24.x` patch releases, I just tested it on something else:
```
versions that meet the requirements `<=0.1` are: 0.1.43, 0.1.42, 0.1.41, 0.1.40, 0.1.39, 0.1.37, 0.1.36, 0.1.35, 0.1.34, 0.1.33, 0.1.32
```